### ISSUE TITLE
Tamas Kocsis @HelloZeroNet is a piece of shit!

### ISSUE THREAD

**Issue by antifa** [HelloZeroNet/ZeroNet#2279]
@HelloZeroNet has closed the pull request where I shipped the new **GPLv3+**! https://github.com/HelloZeroNet/ZeroNet/pull/2270

He also closed this issues which have more than 100+ comments regarding the `LICENSE`: https://github.com/HelloZeroNet/ZeroNet/issues/2265, https://github.com/HelloZeroNet/ZeroNet/issues/2268

I reported the repository to GitHub for various license violations and illegal distribution of the software and this repository currently under investigation: 
```
Hello,

Thanks for taking the time to let us know. 
Our team is currently investigating the repository in question.

Please let us know if we can help in any other way!

Thanks,
```

Currently there is also a voting (https://github.com/HelloZeroNet/ZeroNet/issues/2273) what is locked from me, even than that I was who reported that ZeroNet violates many licenses and other peoples copyright for years.

I'm a contributor to ZeroNet core but Tamas Kocsis **given me no credit for my contribution**, closed pull request copied code from it and merged in **his name**! 

I also noted (https://github.com/HelloZeroNet/ZeroNet/issues/2276) that I will not follow any LICENSE which is less than **GPLv3+** because ZeroNet needs to be liberetaed from this people who currently on the way to effectively destroy the whole project. @imachug Even **tried to claim** yesterday **proprietary rights** over his contributions! I will never remove the **GPLv3+** license from my repository no matter what!

**Comment by mknepper** [comment]
You said you didn't see anything that needed change, which means there is no issue. Therefore, complaint and issue closed.

**Comment by mknepper** [comment]
> I'm a contributor to ZeroNet core but Tamas Kocsis **given me no credit for my contribution**, closed pull request copied code from it and merged in **his name**!

I could be blind, but can you show me where you contributed to HelloZeroNet's repository? I just see your fork and not the main repo.

**Comment by antifa** [comment]
My contributions: 
- https://github.com/HelloZeroNet/ZeroNet/commit/e1f73697ff7cabd55db4e63b6b77099222f1cb4c
- https://github.com/HelloZeroNet/ZeroNet/commit/cb4a4bd7076fbfe760bc88c46d90157c1a19cebf
- https://github.com/HelloZeroNet/ZeroNet/commit/24ba2a150b84ecc8c8525c5c7657738cd843add6
- https://github.com/HelloZeroNet/ZeroNet/commit/e1d92bf0ecb0da3272ee59bb886e623ebc69744d

**Comment by mknepper** [comment]
These are contributed by another user. Do you have an alternate account? If so, you could make that known.

**Comment by antifa** [comment]
I do have many GitHub accounts to be honest with you. Even an organization.

**Comment by mknepper** [comment]
So @shortcutme is your account?

**Comment by FenzaFenz** [comment]
> I do have many GitHub accounts to be honest with you. Even an organization.
Are you claiming to be a creator of Zeronet?
[d28e1cb4a6d3193a39431c316244e7bbd641a883](https://github.com/HelloZeroNet/ZeroNet/commit/d28e1cb4a6d3193a39431c316244e7bbd641a883)

**Comment by antifa** [comment]
No. He claims my contributions in ZeroNet which I listed for you above! He is clearly not the originator.

Now who running the voting about the license namely @krixano and @imachug tries to discredit me but I was who fixed they issues regarding font loading and other things which was for years was problem! https://github.com/HelloZeroNet/ZeroNet/issues/2249

No @FenzaFenz I don't. Please read the above carefully.

**Comment by mknepper** [comment]
@antifa Can you show us proof that those contributions were stolen from you? Perhaps an archived website we can cross-reference and verify?

**Comment by antifa** [comment]
search for recent ghost pull requests!

**Comment by mknepper** [comment]
Can you directly link us? That way we know we're not missing it. I'd like to see exactly what you're seeing. I can't help otherwise.

**Comment by blazercrypter** [comment]
https://boards.4channel.org/g/thread/73426454#p73426613

**Comment by antifa** [comment]
@mknepper 
> Search for recent ghost pull requests! Example: #2248

**Comment by mknepper** [comment]
> @mknepper
> 
> > Search for recent ghost pull requests! Example: #2248

So that was your account?

**Comment by antifa** [comment]
@mknepper Was. Yes. @imachug  @krixano and @shortcutme immediately after the pull request attacked my account. 
When I got back to GitHub, he Tamas Kocsis merged in his own name my commits. I even asked to give me credit but nothing.

Later I was finding that the whole software is distributed illegally on GitHub **violating other peoples licenses and copyrights for years**.

**Comment by 0x6a73** [comment]
@antifa I'd like to remind you that creating new accounts after being banned from Github is a yet another violation of its terms. Also, @HelloZeroNet I think you can just block @antifa on his profile and as far as I know it makes him unable to post in any of your repos. Can someone confirm this?

**Comment by 0x6a73** [comment]
BTW, can the guy from the 4chan thread stop impersonating me? I'm actively watching it. Thanks.

**Comment by antifa** [comment]
@0x6a73 is not your day again :sob: :rofl:

**Comment by 0x6a73** [comment]
@antifa Wdym? Most probably your issue will be closed again.

Also, since you didn't answer it:
As you've stated, the code is now in the public domain, which means that anyone can do whatever they want with it. Including creating a version of that code that's proprietary or non-GPL, right?

**Comment by krixano** [comment]
Just so everyone is aware, GitHub is currently looking into report requests from me on all of his accounts due to his harassment of multiple people on here. Hacktivist would be his other name, and he created around 4 accounts. Additionally, he deleted most of his comments after trying to implement a change in ZeroNet that would overall be bad. However, I have pictures of most things he said, if anyone cares to know about that.

**Comment by 0x6a73** [comment]
@krixano Thanks for archiving these. I didn't know that he deleted these comments.

**Comment by blazercrypter** [comment]
![Screenshot_2019-11-03 Proposed changes in ZeroNet · Issue #2268 · HelloZeroNet ZeroNet](https://user-images.githubusercontent.com/57305009/68129414-ce2f9180-ff19-11e9-9863-e0bc4346cc2d.png)

**Comment by antifa** [comment]
:rofl: Look deep into it! You will find many Tor exit! 

You are a rotten trash @krixano and you should be ashamed of you shitty shady and misleading voting. 

You found nothing. You fixed nothing. For that matter I was who fixed the issue in your latest issue you opened in ZeroNet regarding the fonts in Tor Browser.

I archived everything in the Internet Archive! Just be clear on that!

**Comment by blazercrypter** [comment]
![1572864273950](https://user-images.githubusercontent.com/57305009/68129509-fcad6c80-ff19-11e9-8fbd-b926e8a33532.png)

**Comment by antifa** [comment]
Stop the screenshots! You can archive pages with **Internet Archive** like this: 

https://web.archive.org/save/https://github.com/HelloZeroNet/ZeroNet/issues/2279

**Comment by krixano** [comment]
Just so you know @antifa , from the point ZeroNet's license was invalid, the software had no license and is considered proprietary.

**Comment by 0x6a73** [comment]
Rory, Nov 4, 8:43 AM UTC

Hello, 
Here's an official email from the Github Support Team regarding the first report, that only included info about him breaking the copyright. Three more regarding further issues (such as inciting violence, suicide, threatening, personal attack, racism etc.) are remaining.

Response 1:

> 
> Thanks for contacting us. 
> 
> If you are the copyright holder and want to have the content removed, you have the option of filing a DMCA takedown notice. Instructions, examples, and policy details are available here:  
> https://help.github.com/articles/dmca-takedown-policy/  
> https://help.github.com/articles/guide-to-submitting-a-dmca-takedown-notice/ 
>  
> If you're not the copyright holder, your best bet would be to contact the owner of the content and let them decide if they'd like to take action.
> 
> Thanks,
> Rory
>

**Comment by antifa** [comment]
:rofl: Come send me one DMCA! I want to know your name and full address. :rofl: Also if you send me DMCA you would need to send to another about 2,000 people the same! You are a joke, bro!

**Comment by 0x6a73** [comment]
@antifa Also, are you going to finally answer my question or not?

**Comment by blazercrypter** [comment]
![1572784738685](https://user-images.githubusercontent.com/57305009/68130095-0edbda80-ff1b-11e9-9c57-3e1ed813d182.png)
![1572783585772](https://user-images.githubusercontent.com/57305009/68130106-113e3480-ff1b-11e9-8a64-f04bbdfd3b47.png)
![1572732177708](https://user-images.githubusercontent.com/57305009/68130115-17341580-ff1b-11e9-926b-6f3009a4c267.png)

**Comment by 0x6a73** [comment]
@blazercrypter Thanks man, you're amazing.

**Comment by antifa** [comment]
Also since ZeroNet is not owned by nobody, there should be a voting about the DMCA! :rofl:

**Comment by 0x6a73** [comment]
@antifa We've already presented you arguments why your thinking is invalid and you didn't address them. Learn to read. Also, the voting is here just because the project owner thought that the contributors should decide on it. Normally, @HelloZeroNet could just declare that "I wanna use WTFPL license" and nothing could stop it from doing so if the project is in public domain.

TL;DR: not your project

**Comment by antifa** [comment]
> Just so you know @antifa , from the point ZeroNet's license was invalid, the software had no license and is considered proprietary, including all contributions (in fact, all contributions have copyright regardless of the license unless otherwise given up explicitly by the contributors).

I considering you mom in the open source, so I can put something in it right? Since when Open Source becomes proprietary? If so than why you molesting me and not other about 2,000 people? 

I tell you! Because you are a Hungarian-Polish neo-Nazi trash! Everyone knows that Hungary is full of fascists shitheads just like Poland. ZeroNet is going to be on GPLv3+ or this project can be buried right now.

**Comment by 0x6a73** [comment]
@antifa Then bury it yourself. We're all waiting or it - that is either a lawsuit from you or something else ;)

**Comment by blazercrypter** [comment]
> > Just so you know @antifa , from the point ZeroNet's license was invalid, the software had no license and is considered proprietary, including all contributions (in fact, all contributions have copyright regardless of the license unless otherwise given up explicitly by the contributors).
> 
> I considering you mom in the open source, so I can put something in it right? Since when Open Source becomes proprietary? If so than why you molesting me and not other about 2,000 people?
> 
> I tell you! Because you are a Hungarian-Polish neo-Nazi trash! Everyone knows that Hungary is full of fascists shitheads just like Poland. ZeroNet is going to be on GPLv3+ or this project can be buried right now.

Who's the racist here again?

**Comment by antifa** [comment]
I'm **anti-fascist** you maniac!

**Comment by 0x6a73** [comment]
@blazercrypter Yep, a guy claiming to represent an "antifascist" organization being fascist. How ironic.

**Comment by 0x6a73** [comment]
@antifa North Korea's official name is **Democratic** People's Republic of Korea, but it doesn't make it any more democratic, does it? ;)

**Comment by antifa** [comment]
It is more democratic the the fascist south! The United States also call itself democracy but it really a terrorist  and extreme fascists state like Israel where "Zionists" shooting children in the head nearly every day.
